.gitlab-ci.yml

```diff
@@ -9,15 +9,16 @@ stages:
   - build
 
 variables:
-  GIT_STRATEGY: fetch
-  GIT_SUBMODULE_STRATEGY: recursive
-  GIT_DEPTH: 3
-  CARGO_INCREMENTAL: 0
-  CARGO_HOME: "/ci-cache/${CI_PROJECT_NAME}/cargo/${CI_JOB_NAME}"
-  REGISTRY: registry.parity.io/parity/infrastructure/scripts
+  GIT_STRATEGY:                    fetch
+  GIT_SUBMODULE_STRATEGY:          recursive
+  GIT_DEPTH:                       3
```

Comment:
Something magic about this git depth?

Reply:
not at all, just faster to sync. Actually I will change it later.

Reply:
but why not 1?